extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx

```diff
@@ -342,6 +342,17 @@ export const TransactionDetails = ({
   const { scannedAsset: scannedDestAsset } = useScanAsset(destAssetToScan);
   const isDestAssetSuspicious = isAssetSuspicious(scannedDestAsset);
 
+  // user has refreshed the page and lost needed transaction data. Force them to start over
+  if (amount === "0") {
```

Comment:
slightly unrelated fix; kick the user out of send/swap flow if they refreshed in the middle